es-ds-components/components/es-form-radio-cards.vue

```diff
@@ -0,0 +1,83 @@
+<script setup lang="ts">
+interface IOptions {
+    id: string;
+    text: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    value: any;
+    disabled?: boolean;
+    inline?: boolean;
+}
+
+interface IProps {
+    id: string;
+    label: string;
+    name?: string;
+    options?: IOptions[];
+    inline?: boolean;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    modelValue?: any;
+    hasIcon?: boolean;
+    labelClass?: string;
+    labelSrOnly?: boolean;
+}
+const props = withDefaults(defineProps<IProps>(), {
+    name: '',
+    options: undefined,
+    inline: false,
+    modelValue: undefined,
+    hasIcon: false,
+    labelClass: 'font-size-h3',
+    labelSrOnly: false,
+});
+
+const emit = defineEmits(['update:model-value']);
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function handleUpdate(value: any) {
+    emit('update:model-value', value);
+}
+</script>
+
+<template>
+    <fieldset
+        :id="`${props.id}-fieldset`"
+        class="form-group">
+        <legend
+            :id="`${props.id}-legend`"
+            class="font-size-h1 font-weight-bolder mb-200 pb-0 text-dark"
+            tabindex="-1">
+            <!-- How are these classes getting added?: bv-no-focus-ring col-form-label pt-0 -->
```

Comment:
I think we do need to address this, because there's no focus ring when you navigate by keyboard the way there was with the old one. This is important for accessibility so that people using the site without a mouse can see where they are on the page.